README.md

```diff
@@ -315,6 +315,22 @@ Test.ps1 | This is the PowerShell script to test that the uploaded ISO matches t
 Feel free to modify these files to suit your specific needs.
 General changes that you want to contribute back via a pull request are much appreciated.
 
+## Default GreenlakeAdmin password
+
+These scripts support one of two ways of securing the default GreenlakeAdmin account. This is the built-in Administrator account that has been renamed.
+
+1. Statically defined password in the user-data file.
+   * This will enable a known password to be used for the GreenlakeAdmin account
+   * This password should be set in clear text in the user-data file with the passwd property in the users section
+   * This password must meet password complexity requirements of the Windows OS (default of minimum 10 characters with a mix of upper-case, lower-case, number, and symbol)
+   * In addition, the CloudBase-Init configuration file must load UserDataPlugin at some point after CreateUserPlugin but before LocalScriptPlugin
+   * After the initial boot if Windows and giving CloudBase-Init time to run, it will set this password and you will be able to use it to log in via RDP or Serial Console SAC
```

Comment:
"After the initial boot if Windows and giving CloudBase-Init time to run" - can you please double check this sentence?